src/applications/JobManager.java

```diff
@@ -0,0 +1,60 @@
+package applications;
+
+public class JobManager {
+
+    static int numJobs; // number of jobs
+
+    /**
+     * change the state of theMachine
+     *
+     * @return last job run on this machine
+     */
+    static Job changeState(int theMachine) {// Task on theMachine has finished,
```

Comment:
While I agree that changeState does not belong in MachineShopSimulator, I strongly disagree that it belongs here. changeState is changing the state of the Machine corresponding to theMachine, therefore I would rather this method belong to Machine. I think it is telling that it does not belong here that MachineShopSimulator keeps being referenced to get the machine that's having its state changed and to get the eList (see lines 19, 20, 22, 24 - 28, 33, 34).